ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,328 @@
 #
 # - ``scripts``: configure phpipam-scripts
 #
-phpipam_mode: [ 'webui', 'scripts' ]
+phpipam__mode: [ 'webui', 'scripts' ]
 
                                                                    # ]]]
-# ---- Basic configuration options ----
-
-# .. envvar:: phpipam_dependencies [[[
+# .. envvar:: phpipam__dependencies [[[
 #
 # Should phpipam role manage it's own dependencies?
-phpipam_dependencies: True
+phpipam__dependencies: True
+                                                                   # ]]]
 
+# .. envvar:: phpipam__php__dependent_packages [[[
+#
+# List of PHP packages to install using :ref:`debops.php` role.
+phpipam__php__dependent_packages:
+  - 'mysql'
+  - 'gmp'
+  - 'gd'
+  - 'curl'
+  - 'mcrypt'
+  - 'ldap'
+  - 'php-pear'
                                                                    # ]]]
-# .. envvar:: phpipam_domain [[[
+# .. envvar:: phpipam__domain [[[
 #
 # Domain on which phpIPAM is configured in nginx
-phpipam_domain: [ 'ipam.{{ ansible_domain }}' ]
+phpipam__domain: [ 'ipam.{{ ansible_domain }}' ]
+
+                                                                   # ]]]
+# .. envvar:: phpipam__nginx_auth_realm [[[
+#
+# Webserver authentication realm.
+phpipam__nginx_auth_realm: 'IPAM access is restricted'
 
                                                                    # ]]]
-# .. envvar:: phpipam_nginx_auth_realm [[[
+# .. envvar:: phpipam__nginx_access_policy [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_nginx_auth_realm: 'IPAM access is restricted'
+# Name of the "nginx access policy" for phpIPAM. See ``debops.nginx``
+# Ansible role for more details.
+phpipam__nginx_access_policy: ''
 
                                                                    # ]]]
-# .. envvar:: phpipam_nginx_access_policy [[[
+# .. envvar:: phpipam__database_schema [[[
+#
+# phpIPAM database schema loaded by Ansible
+phpipam__database_schema: '{{ phpipam__git_checkout + "/db/SCHEMA.sql" }}'
+                                                                   # ]]]
+
+# .. envvar:: phpipam__php5_pool [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_nginx_access_policy: ''
+# Configuration for php5-fpm pool
+phpipam__php5_pool:
+  enabled: True
+  name: 'phpipam'
+  user: '{{ phpipam__user }}'
+  group: '{{ phpipam__group }}'
+                                                                   # ]]]
 
+# .. envvar:: phpipam__nginx_upstream_php5 [[[
+#
+# nginx configuration - PHP5 upstream
+phpipam__nginx_upstream_php5:
+  enabled: True
+  name: 'php5_phpipam'
+  type: 'php5'
+  php5: 'phpipam'
                                                                    # ]]]
-# ---- User, group, database configuration ----
 
-# .. envvar:: phpipam_user [[[
+# .. envvar:: phpipam__nginx_server [[[
 #
-# System user and group for phpIPAM application
-phpipam_user: 'phpipam'
+# nginx configuration - phpIPAM server
+phpipam__nginx_server:
```

Comment:
You should put the variables for other roles at the end of the `defaults/main.yml` in their own section.

Reply:
Done